forgerock-openbanking-uk-aspsp-rs/forgerock-openbanking-uk-aspsp-rs-mock-store/forgerock-openbanking-uk-aspsp-rs-mock-store-server/src/main/java/com/forgerock/openbanking/aspsp/rs/store/api/openbanking/payment/v3_1_5/internationalstandingorders/InternationalStandingOrderConsentsApiController.java

```diff
@@ -138,12 +139,13 @@ public ResponseEntity getInternationalStandingOrderConsentsConsentId(
         }
         FRInternationalStandingOrderConsent internationalStandingOrderConsent = isInternationalStandingOrderConsent.get();
 
-        return ResponseEntity.ok(packageResponse(internationalStandingOrderConsent));
+        return ResponseEntity.ok(entityInstance(internationalStandingOrderConsent));
     }
 
-    private OBWriteInternationalStandingOrderConsentResponse7 packageResponse(FRInternationalStandingOrderConsent internationalStandingOrderConsent) {
+    private OBWriteInternationalStandingOrderConsentResponse7 entityInstance(FRInternationalStandingOrderConsent internationalStandingOrderConsent) {
```

Comment:
What was wrong with `packageResponse`? I'm not sure if this new name is any clearer? :-)

Reply:
I thought that was confuse because that method return an instance entity object and I thought that the new name was better, return an entity instance for the response.